contrib/scripts/kind.sh

```diff
@@ -260,6 +259,11 @@ kubectl taint nodes --all node-role.kubernetes.io/control-plane-
 kubectl taint nodes --all node-role.kubernetes.io/master-
 set -e
 
+# Set start of unprivileged port range to 1024
+# Docker defaults to 0
+# https://github.com/moby/moby/pull/41030
+docker ps --filter "label=io.x-k8s.kind.cluster=${cluster_name}" --format "{{.ID}}" | xargs -I container_id docker exec container_id sysctl -w net.ipv4.ip_unprivileged_port_start=1024
```

Comment:
Note that you can also get the kind nodes with `kind get nodes --name $cluster_name`

Reply:
Thanks for chiming in @gandro 

I agree. but this gives me: 

```
❯ kind get nodes --name kind
kind-control-plane
kind-worker
```

whereas i'm interested in the docker container ids (or names) that differ from the cluster names. Hence i'm using the clustername as a filter but output the ID of the container.

Reply:
oh wow, due to line wrapping i saw a wrong column as the name of the container. Indeed it's the name of the kind node itself. should switch to an external monitor :D :whataweek:

Thanks @gandro

Reply:
fixed. at least it was still in draft mode :sweat_smile: